Explorer/TableViewCell.swift

```diff
@@ -0,0 +1,202 @@
+        label.text = "Buddies"
+        return label
+    }()
+    
+    // TODO: Buddies icons
+    let buddyView: UIView = {
+        let view = UIView()
+        
+        for n in 0...4 {
+            let frame = CGRect(x: n * 15, y: 0, width: 30, height: 30)
+            let buddyIconView = UIView(frame: frame)
+            buddyIconView.layer.cornerRadius = 15
+            buddyIconView.backgroundColor = .gray
+            view.addSubview(buddyIconView)
+        }
+        
+        return view
+    }()
+    
+    // MARK: "Duration" label
+    let durationLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .black
+        label.textAlignment = NSTextAlignment.right
+        label.font = UIFont(name: "Roboto-Regular", size: 18)
+        label.text = "DURATION"
+        return label
+    }()
+    
+    // MARK: Duration value label
+    let durationValueLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .black
+        label.textAlignment = NSTextAlignment.right
+        label.font = UIFont(name: "Roboto-Black", size: 20)
+        label.text = "43min"
+        return label
+    }()
+    
+    // MARK: "Depth" label
+    let depthLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .black
+        label.textAlignment = NSTextAlignment.right
+        label.font = UIFont(name: "Roboto-Regular", size: 18)
+        label.text = "DEPTH"
+        return label
+    }()
+    
+    // MARK: Depth value label
+    let depthValueLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .black
+        label.textAlignment = NSTextAlignment.right
+        label.font = UIFont(name: "Roboto-Black", size: 20)
+        label.text = "41ft"
+        return label
+    }()
+    
+    // MARK: User icon
+    let userIconLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont(name: "Roboto-Regular", size: 24)
+        label.textAlignment = NSTextAlignment.center
+        label.textColor = .white
+        label.layer.backgroundColor = UIColor.brown.cgColor
+        label.layer.cornerRadius = 24
+        label.text = "DB"
+        return label
+    }()
+    
+    // MARK: Username
+    let usernameLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .darkGray
+        label.font = UIFont(name: "Roboto-Regular", size: 12)
+        label.text = "d2burke"
+        return label
+    }()
+    
+    // MARK: Timestamp
+    let timestampLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .darkGray
+        label.font = UIFont(name: "Roboto-Regular", size: 12)
+        label.textAlignment = NSTextAlignment.right
+        label.text = "Thursday, July 24, 2015 at 10:15am"
+        return label
+    }()
+
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        self.addSubview(bgImageView)
```

Comment:
You can actually call `addSubview` with out the `self` here. 

Also, it’s good habit to add content to the `contentView` instead of the cell itself. There are some benefits.